nncf/experimental/torch/fx/quantization/quantize_model.py

```diff
@@ -103,3 +109,46 @@ def quantize_impl(
     quantized_model = _disallow_eval_train(quantized_model)
 
     return quantized_model
+
+
+def compress_weights_impl(
```

Comment:
Does `compress_weights` support for models from `torch.export.export`?

Reply:
when trying to pass the `graph_module` from the ExportedModel, to `nncf.compress_weights`, it runs without problems but no compression is applied.

Reply:
Please, open an issue to support this cause.

Reply:
I have opened and tagged Issue #2987 in this PR